packages/grafana-ui/src/components/Text/Text.tsx

```diff
@@ -30,14 +32,18 @@ export const Text = React.forwardRef<HTMLElement, TextProps>(
       )
     );
 
-    return createElement(
+    const tooltipText = ReactDomServer.renderToString(children);
```

Comment:
I think the best way to do this would be to use `innerText` from the element, but for that we need to use some sort of ref to the `createElement`. Another option would be to just pass `children` directly to the `content` of the tooltip, but that could also have some strange scenario where someone does `<h6>Text<h1>H1</h1></h6>` and then the tooltip content is gigantic inside. But maybe its not our problem 🤷